website/docs/sdk-reference/laravel.md

```diff
@@ -0,0 +1,239 @@
+---
+id: laravel
+title: ConfigCat package for Laravel
+description: ConfigCat package for Laravel. Implement feature flags within your PHP Laravel application using ConfigCat.
+---
+
+[![Star on GitHub](https://img.shields.io/github/stars/pod-point/laravel-configcat.svg?style=social)](https://github.com/pod-point/laravel-configcat/stargazers)
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/pod-point/laravel-configcat.svg?style=flat-square)](https://packagist.org/packages/pod-point/laravel-configcat)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/pod-point/laravel-configcat/run-tests.yml?branch=main&label=tests)
+[![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](https://github.com/Pod-Point/laravel-configcat/blob/main/LICENSE.md)
+[![Total Downloads](https://img.shields.io/packagist/dt/pod-point/laravel-configcat.svg?style=flat-square)](https://packagist.org/packages/pod-point/laravel-configcat)
+
+<a href="https://github.com/Pod-Point/laravel-configcat" target="_blank">ConfigCat package for Laravel on GitHub</a>
+
+Implement feature flags within your Laravel application using [ConfigCat](https://configcat.com).
+
+## Installation
+
+You can install the package via composer:
+
+```bash
+composer require pod-point/laravel-configcat
+```
+
+### Publishing the config file
```

Comment:
Hi @clemblanco! Thanks very much for your PR, we are all excited, you made our day :grin: 

Here is a suggestion on the PR:
We like to call this part "options" or "setup" or something similar since the expression "configuration" is overused in the context of ConfigCat. Lots of things here called configs or configurations. I think it is better to make a difference where possible.